homeassistant/components/tibber/sensor.py

```diff
@@ -213,7 +218,14 @@ def model(self):
     @property
     def name(self):
         """Return the name of the sensor."""
-        return f"Real time consumption {self._name}"
+        return f"{self._sensor} {self._name}"
+
+    def set_state(self, state):
+        """Set sensor state."""
+        self._state = state
+        if self.hass is None:
```

Comment:
We can remove this check when we use our dispatch helper to signal the entity. We'll connect the signal in `async_added_to_hass` with `async_dispatcher_connect`.